src/GitHub.VisualStudio/Styles/LinkDropDown.xaml

```diff
@@ -0,0 +1,83 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:cache="clr-namespace:GitHub.VisualStudio.Helpers"
+                    xmlns:ui="clr-namespace:GitHub.UI;assembly=GitHub.UI">
+    
+    <ResourceDictionary.MergedDictionaries>
+        <cache:SharedDictionaryManager Source="pack://application:,,,/GitHub.UI;component/SharedDictionary.xaml" />
+    </ResourceDictionary.MergedDictionaries>
+
+    <Style x:Key="HyperLinkToggleButton" TargetType="ToggleButton">
```

Comment:
No unfortunately not - the `ActionLink` styles apply to a button, which swallow the click and so the toggle button doesn't get toggled.